src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/PersonOASysSectionsTest.kt

```diff
@@ -0,0 +1,215 @@
+                OffenderDetailsSummaryFactory()
+                  .withCrn("CRN")
+                  .withFirstName("James")
+                  .withLastName("Someone")
+                  .produce()
+              )
+            )
+        )
+    )
+
+    val offenceDetails = OffenceDetailsFactory().produce()
+    wiremockServer.stubFor(
+      get(WireMock.urlEqualTo("/offence-details/CRN"))
+        .willReturn(
+          aResponse()
+            .withHeader("Content-Type", "application/json")
+            .withStatus(200)
+            .withBody(
+              objectMapper.writeValueAsString(
+                offenceDetails
+              )
+            )
+        )
+    )
+
+    val roshSummary = RoshSummaryFactory().produce()
+    wiremockServer.stubFor(
+      get(WireMock.urlEqualTo("/rosh-summary/CRN"))
+        .willReturn(
+          aResponse()
+            .withHeader("Content-Type", "application/json")
+            .withStatus(200)
+            .withBody(
+              objectMapper.writeValueAsString(
+                roshSummary
+              )
+            )
+        )
+    )
+
+    val risksToTheIndividual = RiskToTheIndividualFactory().produce()
+    wiremockServer.stubFor(
+      get(WireMock.urlEqualTo("/risk-to-the-individual/CRN"))
+        .willReturn(
+          aResponse()
+            .withHeader("Content-Type", "application/json")
+            .withStatus(200)
+            .withBody(
+              objectMapper.writeValueAsString(
+                risksToTheIndividual
+              )
+            )
+        )
+    )
+
+    val riskManagementPlan = RiskManagementPlanFactory().produce()
+    wiremockServer.stubFor(
+      get(WireMock.urlEqualTo("/risk-management-plan/CRN"))
+        .willReturn(
+          aResponse()
+            .withHeader("Content-Type", "application/json")
+            .withStatus(200)
+            .withBody(
+              objectMapper.writeValueAsString(
+                riskManagementPlan
+              )
+            )
+        )
+    )
+
+    val needsDetails = NeedsDetailsFactory().apply {
+      withAssessmentId(34853487)
+      withAccommodationIssuesDetails("Accommodation", true, false)
+      withAttitudeIssuesDetails("Attitude", false, true)
+      withFinanceIssuesDetails(null, null, null)
+    }.produce()
+
+    wiremockServer.stubFor(
+      get(WireMock.urlEqualTo("/needs-details/CRN"))
+        .willReturn(
+          aResponse()
+            .withHeader("Content-Type", "application/json")
+            .withStatus(200)
+            .withBody(
+              objectMapper.writeValueAsString(
+                needsDetails
+              )
+            )
+        )
+    )
+
+    val jwt = jwtAuthHelper.createAuthorizationCodeJwt(
+      subject = "username",
+      authSource = "delius",
+      roles = listOf("ROLE_PROBATION")
+    )
+
+    webTestClient.get()
+      .uri("/people/CRN/oasys/sections?selected-sections=11&selected-sections=12")
```

Comment:
it's interesting how Spring / OpenAPI handle this repeated url param and know to accumulate the values into a list...

Reply:
I tested this in Postman too - it seems happy with either:
- `key=value1&key=value2`
- `key=value1,value2

But wouldn't take `key=value1|value2`